packages/shorebird_cli/pubspec.yaml

```diff
@@ -26,11 +26,7 @@ dependencies:
   cli_util: ^0.4.1
   collection: ^1.18.0
   crypto: ^3.0.3
-  googleapis_auth:
-    git:
-      url: https://github.com/shorebirdtech/googleapis.dart.git
-      path: googleapis_auth
-      ref: 647f96d5ffdae63e41aeea091c34acfffff533e7
+  googleapis_auth: ^1.6.0
```

Comment:
Don't we also need to bump the min dart sdk version?

Reply:
nevermind just checked and it looks like googleapis_auth min is 3.0.0 so we're good 👍

Reply:
I think there may a dependency of that project that requires a higher version. Checking with an older Dart version now.

Reply:
It looks like several of our dependencies require newer versions of Dart than our pubspec currently says it requires. `http 1.2.1` requires Dart 3.3.0, for example.